src/user/models/auth/auth.service.ts

```diff
@@ -45,11 +49,13 @@ export class AuthService {
     }
 
     const user = await this.userService.createUser(dto, userIp, ref?.origin);
-    return { accessToken: this.generateToken(user) };
+    return { accessToken: this.generateUserToken(user) };
   }
 
-  async signIn({ address, signature }: AuthCredentialsDto): Promise<{ accessToken: string }> {
-    const user = await this.userRepo.getByAddress(address);
+  async signIn({ address, signature }: AuthCredentialsDto, companyLogin = false): Promise<{ accessToken: string }> {
```

Comment:
I think I would create a different method for the companySignIn. There's not much shared logic. And getting a user from a walletRepo is kind of confusing 😉